rc/base/ctags.kak

```diff
@@ -16,8 +16,9 @@ def -params 0..1 \
         for tags in "$@"
         do
             [ -f "$tags" ] || continue
-            namecache="$(dirname "$tags")"/.kak."$(basename "$tags")".namecache
-            if [ "$namecache" -ot "$tags" ]; then
+            namecache="${tags%/*}"/.kak."${tags##*/}".namecache
```

Comment:
Actually there are some cases that `dirname` handles that are not replaceable with a shell expansion (like returning `.` when the argument is a simple filename), maybe this first subshell should have stayed?